### PR TITLE
Update ResultsDocumentGeneration.R

### LINKED ISSUE #92: Bad formatting for output docx document
Hi everyone,
I'm having an issue with the version 1.1.0 of the package running on R 4.2.2.

When generating the docx output document the chapter 4.6 (mapped codes) is empty and all its content is at the bottom of the document, after the chapter 9 (checklist).

### PR DESCRIPTION
**Pull request by MaximMoinat**
fixes #92

### CODE REVIEW

**Review by leeevans — APPROVED** [review]

**Comment by leeevans** [comment]
This is an important pull request to fix the format of the generated report.
Can someone with write access to this repo please review and approve it?

**Comment by MaximMoinat** [comment]
Thanks @leeevans for your review and pushing this.
I have requested @PRijnbeek for a final review, after which I will merge and create a new release.

**Pull request merged**
